cloudify_types/cloudify_types/component/component.py

```diff
@@ -232,7 +232,7 @@ def _set_secrets(self):
         for secret_name in self.secrets:
             self._http_client_wrapper('secrets', 'create', {
                 'key': secret_name,
-                'value': self.secrets[secret_name],
+                'value': str(self.secrets[secret_name]),
```

Comment:
this is most likely gonna break non-ascii secrets in py2
for a 2+3-compat version you can use:
```python
u'{0}'.format(self.secrets[secret_name])
```